.github/workflows/check-generated.yml

```diff
@@ -13,7 +13,7 @@ permissions:
   contents: read
 
 jobs:
-  check-mocks:
+check-mocks:
```

Comment:
Various CI workflows aren't actually running. See errors in yaml files here: https://github.com/celestiaorg/celestia-core/pull/792/checks?check_run_id=7488437122

Reply:
thanks for pointing this out, I should have checked since this is what happened last time. I wish github wouldn't give me the :heavy_check_mark:  if tests aren't running!